src/components/ConfirmationModal.tsx

```diff
@@ -69,12 +60,8 @@ const ConfirmationModal = (props: ConfirmationModalPropsType): JSX.Element => {
             <CloseIcon />
           </Icon>
           <Message>{text}</Message>
-          <Button onPress={closeModal} buttonTheme={BUTTONS_THEME.dark}>
-            <Label light>{cancelButtonText}</Label>
-          </Button>
-          <Button onPress={confirmationAction} buttonTheme={BUTTONS_THEME.light}>
-            <Label light={false}>{confirmationButtonText}</Label>
-          </Button>
+          <Button label={cancelButtonText} onPress={closeModal} buttonTheme={BUTTONS_THEME.dark} />
```

Comment:
Yeah, well using `BUTTONS_THEME.dark` makes it possible to refactor it. E.g. if right-clicking in Intellij and do refactor, all occurences are renamed. This will not work when using the string. Think we should in general make terms which way we prefer.